test/unit/runner.spec.js

```diff
@@ -452,9 +451,29 @@ describe('Runner', function() {
       'at processImmediate [as _immediateCallback] (timers.js:321:17)'
     ];
 
+    before(function() {
+      // Only for Node running on Windows
+      if ('platform' in process && process.platform === 'win32') {
+        var addDrive = function(str) {
+          var drive = 'C:';
+          var pos = str.indexOf(path.posix.sep);
+          return pos !== -1 ? str.slice(0, pos) + drive + str.slice(pos) : str;
+        };
+
+        var useWinPathSep = function(str) {
+          return str.split(path.posix.sep).join(path.win32.sep);
+        };
+
+        // Fake Windows pathnames in stacktrace
+        stack = stack.map(function(line) {
+          return useWinPathSep(addDrive(line));
+        });
+      }
+    });
+
     describe('shortStackTrace', function() {
-      beforeEach(function() {
-        if (path.sep !== '/') {
+      before(function() {
+        if ('browser' in process) {
```

Comment:
personally I'd prefer `if (process.brower) {`

Reply:
Since we've found that this test was skipped in browser testing unintentionally, should it be turned on instead of continue skipping?

Reply:
It doesn't currently work.
https://travis-ci.org/mochajs/mocha/jobs/486574106#L1594

Reply:
> personally I'd prefer `if (process.brower) {`

Lacking an [existential operator](https://esdiscuss.org/topic/the-existential-operator), IMO this style better communicates the fact that it _may not exist_.

Reply:
Actually, [it does](https://github.com/defunctzombie/node-process/blob/77caa43cdaee4ea710aa14d11cea1705293c0ef3/browser.js#L156)...